observability-charter.md

```diff
@@ -0,0 +1,207 @@
+
+A full list of [CNCF projects][projs] can be found at [landscape.cncf.io].
+
+[cncf]: https://www.cncf.io
+[projs]: https://www.cncf.io/projects
+[landscape.cncf.io]: https://landscape.cncf.io
+[sig]:  https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md
+[cn-def]: https://github.com/cncf/toc/blob/master/DEFINITION.md
+
+## Mission
+
+Consistent with the CNCF [SIG] definition, the mission of SIG Observability
+is to:
+
+- Foster and grow the ecosystem of observability related projects, users, and
+  maintainers.
+- Identify and report gaps in the CNCF's project portfolio on topics of
+  observability to the TOC and the wider CNCF community.
+- Collect, curate, champion, and disseminate patterns and current best practices
+  related to the observation of cloud-native systems that are effective and
+  actionable.
+- Educate and inform users with unbiased, accurate, and pertinent information.
+- Educate and help other CNCF projects in regarding observability techniques and
+  practices available within the CNCF.
+- Provide and maintain a vendor-neutral venue for relevant thought validation,
+  discussion, and project feedback.
+- Provide a ladder for community members to become involved with the technical
+  oversight of projects within the SIG's scope in an open, transparent, and
+  inclusive way.
+
+## Areas considered in Scope
+
+Observability focuses on patterns, projects, tools, and techniques related to
+topics such as:
+
+- Methodologies for instrumenting, collecting, processing, storing, querying,
+  curating, and correlating observational data such as metrics, logging/events,
+  trace spans, and profiling of cloud native workloads.
+- Using distributed trace tooling to observe a series of calls between
+  microservices to understand where time is being spent.
+- Managing the complexity, operational cost, and resource consumption of
+  observability tools and systems at the enterprise scale.
+- Best practices for meaningful alerting, queries, and operational dashboards
+  including how to manage things including rules, definitions, thresholds and
+  policies.
+- How developers, operators, SRE, IT, and other actors comprehend, process, and
+  reason on distributed cloud-native systems.
+- Projects that incorporate novel & insightful approaches to utilizing
+  observability data such as:
+  - ML, model training, Bayesian networks, and other data science techniques
+    that enable anomaly & intrusion detection.
+  - correlating resource consumption with costing data to reduce the total cost
+    of cloud native infrastructure
+  - Using observability data exposed by service meshes, orchestrators, and other
+    metric sources to inform continuous deployment tooling (e.g. Canary
+    Predicates/Judges).
+- Objective curation and generation of case studies pertaining to delivering
+  observability tools/systems to end users.
+- Best practices around observability and its continuous improvement, e.g. post
```

Comment:
I think this might be the same as ` Best practices for meaningful alerting, queries, and operational dashboards
  including how to manage things including rules, definitions, thresholds and
  policies.`

Maybe we can merge those to be more concise? (:

Reply:
If memory serves from discussion last week the distinction here is/was a focus on alerting/operational dashboards, vs. installing and configuring the systems themselves.  WDYT?  Agreed we could probably merge this to a common clause.

Reply:
Cool, let's keep like it is for now then, iterate later (: